### PR TITLE
improve tutorial, add library overview

### PR DESCRIPTION
**Pull request by dbu**
This is now ready. The doc homepage looks like this:
doc homepage

### CODE REVIEW

File: source/_guides/2-create-an-organization.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+```php
+/** @var \Rokka\Client\User $userClient */ 
+$userClient->createOrganization('testorganization', 'billing@testorganization.com', 'Test Organization Ltd.');
+```
+
+This creates an organization called "testorganization" and returns a json structure that reflects the submitted data and adds the unique id of your organization. 
+
+```javascript
+{
+    "id": "c03683b067927d77973b458e0baa40aa7b5e5418",
```
Comment (dbu) [inline source/_guides/2-create-an-organization.md]: what is the use of uuid of anything except the hash for the uploaded image?

File: source/_guides/3-dynamic-rendering.md [hunk 2]
```diff
@@ -1,26 +0,0 @@
----
-title: Dynamic rendering
```
Comment (dbu) [inline source/_guides/3-dynamic-rendering.md]: eliminated this from the turial. mentioning in the rendering chapter but then pointing to reference

File: source/_views/library.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{% extends "guide.html" %}
+
+{% block article_title %}Rokka Client Libraries{% endblock %}
+
+{% block articles %}
+    {% include 'article_list' with { 'articles': data.libraries } %}
```
Comment (dbu) [inline source/_views/library.html]: this seems to work

File: source/_guides/2-create-an-organization.md (hunk 1 above)
Comment (ChristianRiesen) [inline source/_guides/2-create-an-organization.md]: UUIDs are internal identifiers only, needed for linking users to organizations. Can also be helpful for debugging.

File: source/documentation/index.html [hunk 4]
```diff
@@ -13,20 +13,23 @@
             <div href="#" class="featurebox transition">
                 <h2 class="featurebox__title h2 m-0 transition">Guide</h2>
                 <div class="featurebox__text transition">
-                    <p>{% include 'article_list' with { 'articles': data.guides } %}</p>
+                    {% include 'article_list' with { 'articles': data.guides } %}
+                </div>
+
+                <h2 class="featurebox__title h2 m-0 transition">API Client Libraries</h2>
+                <div class="featurebox__text transition">
+                    {% include 'article_list' with { 'articles': data.libraries } %}
```
Comment (ChristianRiesen) [inline source/documentation/index.html]: As reminder: Look at the "use" in the page header.

Comment (dbu) [inline source/documentation/index.html]: thanks, fixed

**Pull request merged**